plenum/test/consensus/helper.py

```diff
@@ -0,0 +1,70 @@
+from typing import Optional, List
+
+from plenum.common.config_util import getConfig
+from plenum.common.event_bus import InternalBus
+from plenum.server.consensus.replica_service import ReplicaService
+from plenum.server.database_manager import DatabaseManager
+from plenum.server.ledgers_bootstrap import LedgersBootstrap, IN_MEMORY_LOCATION
+from plenum.server.node import Node
+from plenum.server.request_managers.read_request_manager import ReadRequestManager
+from plenum.server.request_managers.write_request_manager import WriteRequestManager
+from plenum.test.greek import genNodeNames
+from plenum.test.helper import MockTimer
+from plenum.test.simulation.sim_network import SimNetwork
+from plenum.test.simulation.sim_random import SimRandom, DefaultSimRandom
+from plenum.test.testing_utils import FakeSomething
+
+
+class TestLedgersBootstrap(LedgersBootstrap):
+    def create_bls_bft(self):
+        return FakeSomething()
+        # self.bls_key_register = bls_key_register
+        # self.bls_crypto_signer = bls_crypto_signer
+        # self.bls_crypto_verifier = bls_crypto_verifier
+        # self.bls_store = bls_store
+
+    def update_txn_with_extra_data(self, txn):
+        return txn
+
+
+def create_test_write_req_manager(name: str) -> WriteRequestManager:
+    db_manager = DatabaseManager()
+    write_manager = WriteRequestManager(db_manager)
+    read_manager = ReadRequestManager()
+    bootstrap = TestLedgersBootstrap(
+        write_req_manager=write_manager,
+        read_req_manager=read_manager,
+        action_req_manager=FakeSomething(),
+        name=name,
+        config=getConfig(),
+        data_location=IN_MEMORY_LOCATION,
+        genesis_dir=IN_MEMORY_LOCATION,
+        ledger_ids=Node.ledger_ids
+    )
+    bootstrap.init_ledgers()
```

Comment:
Is it the main (and the only) interface method of `LedgersBootstrap` that must be called explicitly? If so, I would suggest to make all other methods in `LedgersBootstrap` protected.